### ISSUE TITLE
Any layers for modulo operation?

### ISSUE THREAD

**Issue by prclibo** [BVLC/caffe#2560]
Hi,

I would like to do regression on for example the angle value. So I need some modulo operation layer to make the loss between, say, 1deg and 359deg small. I think I didn't find similar layers implemented in caffe (Correct me if I missed it). So if you think this layer is useful for caffe. I might do a PR and would like to ask for suggestions from caffe admins.

Thank you!

**Comment by shelhamer** [comment]
No, there are not currently layers for this, but adding an activation layer for trig functions seems like it would be useful from time to time.
